lib/buildkite/builder/pipeline.rb

```diff
@@ -57,7 +57,7 @@ def upload
             abort
           end
           logger.info "+++ :toolbox: Setting job meta-data to #{Buildkite.env.job_id.color(:yellow)}"
-          Buildkite::Pipelines::Command.meta_data!(:set, Builder::META_DATA.fetch(:job), Buildkite.env.job_id)
+          Buildkite::Pipelines::Command.meta_data!(:set, Builder::META_DATA.fetch(:job), Buildkite.env.step_key)
```

Comment:
I think the key needs to have the step key in it. So that multiple BKB steps can be unique.

Reply:
So that if a BKB step attempts to upload again, it’ll be the same key and it’ll be already set.